sweep.py

```diff
@@ -151,12 +146,8 @@ def Bs(ix):
     MK = csr_matrix(-omega**2*M+K)
     eplo_p = spsolve(MK,F)
 
-    #Add back zeros (boundaries)
```

Comment:
By addition of these clamping point, total shape of dispolacement matrix seems like disaligned.
To Kai: When we arrange and solve this disalignment, I think I can apply clamp point?
(Sorry, I am utilizing this code withou clamp point)